docs/triage.md

```diff
@@ -2,34 +2,52 @@
 
 As we get more issues and pull requests opened on the GitHub CLI, we've decided on a weekly rotation
 triage role. The initial expectation is that the person in the role for the week spends no more than
-1-2 hours a day on this work; we can refine that as needed. Below is a basic timeline for a typical
-triage day.
+2 hours a day on this work; we can refine that as needed.
 
-1. Note the time
-2. Open every new [issue](https://github.com/cli/cli/issues?q=is%3Aopen+is%3Aissue)/[pr](https://github.com/cli/cli/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse) in a tab
-3. Go through each one and look for things that should be closed outright (See the PR and Issue section below for more details.)
-4. Go through again and look for issues that are worth keeping around, update each one with labels/pings
-5. Go through again and look for PRs that solve a useful problem but lack obvious things like tests or passing builds; request changes on those
-6. Mark any remaining PRs (i.e. ones that look worth merging with a cursory glance) as `community` PRs and move to Needs Review
-7. Look for [issues](https://github.com/cli/cli/issues?q=is%3Aopen+is%3Aissue) and [PRs](https://github.com/cli/cli/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+sort%3Aupdated-desc) updated in the last day and see if they need a response.
-8. Check the clock at each step and just bail out when an hour passes
+## Expectations for incoming issues
 
-# Incoming issues
+All incoming issues need either an **enhancement** or **bug** label.
```

Comment:
Could we see `docs` as a potential 3rd category? When an issue is about improving documentation, in my head it's sometimes both an "enhancement" and a "bug fix" (if the documentation was misleading), but for some cases it feels like it's neither.

Reply:
good idea.